Obj-C/Podfile

```diff
@@ -1,14 +1,14 @@
-platform :ios, '12.0'
+platform :ios, '13.0'
 workspace 'BidMachine-iOS-Examples.xcworkspace'
 
 source 'https://github.com/appodeal/CocoaPods.git'
```

Comment:
Please change the source to `https://github.com/bidmachine/CocoaPods-Specs.git`

Reply:
✅